tests/dashbio_demos/dash-igv/app.py

```diff
@@ -0,0 +1,162 @@
+import os
+import dash_html_components as html
+import dash_core_components as dcc
+from dash.dependencies import Input, Output
+import dash_bio
+
+try:
+    from layout_helper import run_standalone_app
+except ModuleNotFoundError:
+    from .layout_helper import run_standalone_app
+
+text_style = {
+    'color': "#506784",
+    'font-family': 'Open Sans'
+}
+
+_COMPONENT_ID = 'igv-chart'
+
+HOSTED_GENOME_DICT = [
+    {'value': 'hg38', 'label': 'Human (GRCh38/hg38)'},
+    {'value': 'hg19', 'label': 'Human (CRCh37/hg19)'},
+    {'value': 'hg18', 'label': 'Human (hg18)'},
+    {'value': 'mm10', 'label': 'Mouse (GRCm38/mm10)'},
+    {'value': 'rn6', 'label': 'Rat (RGCS 6.0/rn6)'},
+    {'value': 'gorGor4', 'label': 'Gorilla (gorGor4.1/gorGor4)'},
+    {'value': 'panTro4', 'label': 'Chimp (SAC 2.1.4/panTro4)'},
+    {'value': 'panPan2', 'label': 'Bonobo (MPI-EVA panpan1.1/panPan2)'},
+    {'value': 'canFam3', 'label': 'Dog (Broad CanFam3.1/canFam3)'},
+    {'value': 'ce11', 'label': 'C. elegans (ce11)'}
+]
+
+
+def description():
+    return 'A high-performance genomics viewer with an interactive UI and support for a ' \
+           'wide variety of data types and features.'
+
+
+def header_colors():
+    return {
+        'bg_color': '#0F5BA7',
+        'font_color': 'white',
+    }
+
+
+def layout():
+    return html.Div(id='igv-body', className='app-body', children=[
+        html.Div(id='igv-control-tabs', className='control-tabs', children=[
+            dcc.Tabs(
+                id='igv-tabs',
+                value='what-is',
+                children=[
+                    dcc.Tab(
+                        label='About',
+                        value='what-is',
+                        children=html.Div(className='control-tab', children=[
+                            html.H4(className='what-is', children='What is IGV?'),
+                            html.P(
+                                """
+                                The Dash IGV component is a high-performance genomics
+                                data visualization component developed originally by the [IGV
+                                Team]( based at UC San Diego and the Broad Institute. It offers
```

Comment:
This looks like it's trying to be markdown, but it's missing the link and closing paren... could turn this block into `dcc.Markdown`, that would also make the link in the paragraph below active.

Reply:
Fixed in 0e650d0.